ci-operator/config/operator-framework/community-operators/OWNERS

```diff
@@ -0,0 +1,13 @@
+# DO NOT EDIT; this file is auto-generated using tools/populate-owners.
+# from https://github.com/operator-framework/community-operators/blob/06bcdfd43a3f166c7cbb2c42f11b694a72ac4e5f/OWNERS
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- aravindhp
+- awgreene
```

Comment:
Please replace `awgreene` with `kevinrizza`

Reply:
This file gets generated automatically. So if we want that to happen, then community-operators needs to be updated.